File: src/main/java/com/blogly/blogly/domain/post/TitleAlreadyExistsException.java
package com.blogly.blogly.domain.post;

import com.blogly.blogly.domain.exception.DomainException;

public class TitleAlreadyExistsException extends DomainException {

    public TitleAlreadyExistsException(String title) {
        super("There is already a post with title: " + title);
    }
}

File: src/main/java/com/blogly/blogly/domain/post/exception/TitleAlreadyExistsException.kt
package com.blogly.blogly.domain.post.exception

import com.blogly.blogly.domain.exception.DomainException
import com.blogly.blogly.domain.post.Title

class TitleAlreadyExistsException(title: Title) : DomainException("There is already a post with title: ${title.value}")
